### PR TITLE
Normative: Add Temporal[@@toStringTag].

### PR DESCRIPTION
**Pull request by Ms2ger**
Fixes #1539.

### CODE REVIEW

File: spec/temporal.html [hunk 1]
```diff
@@ -14,6 +14,15 @@ <h1>The Temporal Object</h1>
     <li>does not have a [[Call]] internal method; it cannot be invoked as a function.</li>
   </ul>
 
+  <emu-clause id="sec-value-properties-of-the-temporal-object">
+    <h1>Value Properties of the Temporal Object</h1>
+    <emu-clause id="sec-temporal-@@tostringtag">
+      <h1>Intl[ @@toStringTag ]</h1>
```
Comment (ljharb) [inline spec/temporal.html:20]:
```suggestion
      <h1>Temporal [ @@toStringTag ]</h1>
```

**Review by ljharb — CHANGES_REQUESTED** [review]

File: spec/temporal.html (hunk 1 above)
Comment (Ms2ger) [reply spec/temporal.html:20]: Doh.

**Review by ljharb — APPROVED** [review]

**Review by ptomato — APPROVED** [review]
I will mark this as "draft" so it's not accidentally merged before being presented to plenary.

**Comment by ptomato** [comment]
If we go with Temporal.now[@@toStringTag] as discussed on #1539 then this PR will need to be updated.

**Comment by ptomato** [comment]
I believe this is covered by the consensus achieved at the July 2021 TC39 meeting as part of the "Guidance on nested namespaces" item. I will add the updates to `Temporal.Now` as per that item and un-draft this.

**Pull request merged**